docs/bot/synapse_module.md

```diff
@@ -7,10 +7,9 @@ sidebar_label: Synapse module
 
 :::warning
 
-The Synapse module is unmaintained, is exactly the same as the module
-provided by Mjolnir, and will likely be removed from Draupnir
-in the future. However, should the module be removed, mjolnir's
-module will still be compatible with Draupnir's policy lists.
+The Synapse module that was inherited from Mjolnir has been removed from the project.
+The module was unmaintained and untouched for its whole lifetime in Draupnir.
+However the mjolnir module is still compatible with Draupnir's policy lists.
```

Comment:
```suggestion
The version of the Synapse module in the Draupnir repository that was inherited from Mjolnir has been removed.
The module was unmaintained and untouched for its whole lifetime in Draupnir, and we always directed
users to the Mjolnir version, which still exists and is still compatible with Draupnir's policy lists.
```

Reply:
This sounds good to the kitty and communicates the spirit a lot better. Also while we are at it we could probably switch back this box to a info box.